### ISSUE TITLE
Remove "Send" from website footers

### ISSUE THREAD

**Issue by jvillalobos** [mozilla/addons-frontend#9649]
[Firefox Send](https://send.firefox.com/) has been inactive for a while and we don't expect it to come back soon. Let's remove the link from all AMO footers.

**Comment by jvillalobos** [comment]
Mentor: @bobsilverberg

**Comment by rochisha0** [comment]
Can I work on this?

**Comment by jvillalobos** [comment]
@rochisha0 Go ahead!

**Comment by ioanarusiczki** [comment]
@bobsilverberg 
`Send` has been removed from the footer on AMO dev - frontend. I verified with FF80(Win10). 

![no Send](https://user-images.githubusercontent.com/33448286/93876441-89524c00-fcdf-11ea-9cfc-2b8eea400e8e.png)

The Dev Hub footer still displays Send so let me know if I should file an issue on the server side too. 

![the dev hub send](https://user-images.githubusercontent.com/33448286/93876622-cfa7ab00-fcdf-11ea-87d6-5e032b3e4f08.png)
